ngraph/test/frontend/paddlepaddle/test_models/gen_scripts/generate_top_k.py

```diff
@@ -0,0 +1,42 @@
+#
+# top_k paddle model generator
+#
+import numpy as np
+from save_model import saveModel
+import paddle as pdpd
+import paddle.fluid as fluid
+import sys
+
+data_type = 'float32'
+
+
+def top_k(name: str, x, k: int):
+
+    pdpd.enable_static()
+
+    with pdpd.static.program_guard(pdpd.static.Program(), pdpd.static.Program()):
+        node_x = pdpd.static.data(name='x', shape=x.shape, dtype='float32')
+        value, indices = fluid.layers.topk(node_x, k=k, name="top_k")
```

Comment:
thanks for add test case for legacy topk. Can we merge it with topk_v2 into a single test file?